aws/cloud/iam.rego

```diff
@@ -1737,4 +1737,88 @@ iam_deprecated_policies_metadata := {
     "Resource Type": "",
     "Policy Help URL": "",
     "Resource Help URL": "https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.Client.list_attached_user_policies"
-}
\ No newline at end of file
+}
+
+#
+# PR-AWS-CLD-IAM-048
```

Comment:
this policy "PR-AWS-CLD-IAM-048" is referencing missing/incorrect masterSnapshotId "TEST_IAM_08" at master-compliance-test.json